docs/index.md

```diff
@@ -1,44 +1,82 @@
-# Integration Guide
+# Integration guide
 
-## API Key
+Pulse displays metrics that provide insights about the current and historic performance of your software delivery process. To calculate these metrics, Pulse must collect information from key events of your particular software development workflow.
 
-The API key identifies your organization and authorizes you to push events to Pulse.
+Currently, the recommended way of integrating your workflow with Pulse is by using a CLI to send the necessary data to Pulse. Alternatively, you may also use a webhook for the same purpose.
 
-## Pushing Data
+!!! important
+    Before setting up the integration with Pulse, make sure that you have an API key that identifies your organization and authorizes you to send data to Pulse.
 
-Currently, we support two ways to push data: a CLI and an HTTP API.
+To set up the integration with Pulse you must complete these main steps:
 
-### CLI
+1.  **Install the Pulse CLI**
 
-#### Install
+    Download the latest version of the CLI.
 
-The CLI is distributed in a binary format (for multiple operating systems and architectures)
-that you can download:
+1.  **Push data to Pulse**
 
-```sh
-curl -fsSL "https://dl.bintray.com/codacy/pulse/event-cli/<VERSION>/<ARCH>/pulse-event-cli" -o event-cli
-```
+    Send information about key events of your software development process using either the CLI or the webhook.
 
-Use the latest version to replace `<VERSION>`: [ ![Download](https://api.bintray.com/packages/codacy/pulse/event-cli/images/download.svg)](https://bintray.com/codacy/pulse/event-cli/_latestVersion)
+The next sections include detailed instructions on how to complete each step of the integration setup process.
 
-Supported operating systems and architectures to replace `<ARCH>`:
+## 1. Installing the Pulse CLI
 
--  pulse-event-cli_darwin_amd64
--  pulse-event-cli_linux_386
--  pulse-event-cli_linux_amd64
--  pulse-event-cli_windows_386
--  pulse-event-cli_windows_amd64
+Download the latest version of the CLI for your operating system and make sure that you're able to run the binary.
 
-Don't forget to make the binary executable and check that it works:
+1.  Take note of the latest version of the CLI:
```

Comment:
It would be great if we could simplify this part by running a single command, like we do for example on the [codacy-coverage-reporter](https://github.com/codacy/codacy-coverage-reporter#setup).

If we had a Bintray Premium repository we could already simplify a lot by [calling the API endpoint that returns the latest binary version](https://www.jfrog.com/confluence/display/BT/Bintray+REST+API#BintrayRESTAPI-DynamicDownload).

I even considered providing a small script to automatically find the latest version, but this wouldn't work on Windows:

```sh
latest=$(curl -fsSL https://api.bintray.com/packages/codacy/pulse/event-cli/versions/_latest | jq -r '.name') && \
curl -fsSL -o event-cli "https://dl.bintray.com/codacy/pulse/event-cli/$latest/pulse-event-cli_darwin_amd64/pulse-event-cli" && \
chmod +x event-cli && ./event-cli
```

Reply:
that'd be nice.
we probably can use versions as tag and override them every time we publish or something like that.

you probably can also do the same for the OS and architecture.

Reply:
Not sure what is the difference on premium, but this is a feature. It is just a matter of us deciding on the requirements and we can then think of how.